app/src/main/java/com/github/polypoly/app/map/MapActivity.kt

```diff
@@ -69,7 +87,9 @@ class MapActivity : ComponentActivity() {
     private val initialZoom = 18.0
     private val markerSideLength = 100
 
-    private val markerToLocalization = mutableMapOf<Marker, Localization>()
+    private val maxCloseLocationDistance = 10.0
```

Comment:
Private constants such as this one (or above) would perhaps be more expressive if named using capital snake-case and put into the class' companion object (kinda like static constants in other languages).

Reply:
Addressed!